components/screens/post/PostScreen.tsx

```diff
@@ -33,20 +42,23 @@ function PostScreen({
   route: any;
   navigation: NativeStackNavigationProp<any>;
 }) {
+  const [showLoadAll, setShowLoadAll] = useState(true);
```

Comment:
We will move these into the usePost hook. I'll deal with that.

Since we put our useEffect inside of the post hook, it will be cleaner there. We'll also move the navigation.setOptions() to the useEffect in the hook as well.